src/mumble/CMakeLists.txt

```diff
@@ -454,18 +466,21 @@ else()
 
 		target_sources(mumble
 			PRIVATE
-				"GlobalShortcut_unix.cpp"
-				"GlobalShortcut_unix.h"
 				"Log_unix.cpp"
 				"os_unix.cpp"
 		)
 
 		if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
 			find_library(LIB_RT rt)
 			target_link_libraries(mumble PRIVATE ${LIB_RT})
-		endif()
 
-		target_link_libraries(mumble PRIVATE X11::Xext)
+			# TODO: check on evdev/udev or linux/input.h presence instead of Linux as it's also present in FreeBSD
```

Comment:
Just marking this to not overlook it in the course of further processing of this PR :point_up: